executor/index_merge_reader.go

```diff
@@ -660,7 +724,13 @@ func (w *indexMergeTableScanWorker) pickAndExecTask(ctx context.Context) (task *
 		case <-w.finished:
 			return
 		}
+		atomic.AddInt64(&w.stats.WaitTime, int64(time.Since(waitStart)))
+		execStart := time.Now()
 		err := w.executeTask(ctx, task)
+		if w.stats != nil {
+			atomic.AddInt64(&w.stats.FetchRow, int64(time.Since(execStart)))
+			atomic.AddInt64(&w.stats.TableTaskNum, 1)
+		}
```

Comment:
```suggestion
		execStart := time.Now()
		err := w.executeTask(ctx, task)
		if w.stats != nil {
			atomic.AddInt64(&w.stats.WaitTime, int64(execStart.Sub(waitStart)))
			atomic.AddInt64(&w.stats.FetchRow, int64(time.Since(execStart)))
			atomic.AddInt64(&w.stats.TableTaskNum, 1)
		}
```